lib/rules/indent.js

```diff
@@ -97,7 +97,7 @@ module.exports = {
                 const: DEFAULT_VARIABLE_INDENT
             },
             outerIIFEBody: null,
-            MemberExpression: 1
+            MemberExpression: null
```

Comment:
@nzakas why there is a `null` value for `outerIIFEBody`? Should we change that too in another PR?